src/libraries/System.Private.CoreLib/src/System/Collections/ObjectModel/ReadOnlyCollection.cs

```diff
@@ -37,10 +58,10 @@ public void CopyTo(T[] array, int index)
             list.CopyTo(array, index);
         }
 
-        public IEnumerator<T> GetEnumerator()
-        {
-            return list.GetEnumerator();
-        }
+        public IEnumerator<T> GetEnumerator() =>
```

Comment:
Is there a chance that the calling code depends on the exact type of the enumerator returned?

Reply:
I'm sure someone somewhere is, but I expect it's a very rare case, and it's something we say is unsupported. I can mark it as breaking if you think it's worthwhile.

Reply:
I do not think we need to mark this as breaking. I just wanted to highlight as a potential break.